packages/gatsby-transformer-sharp/src/extend-node-type.js

```diff
@@ -12,6 +13,16 @@ const {
   responsiveResolution,
 } = require(`gatsby-plugin-sharp`)
 
+const ImageFormatType = new GraphQLEnumType({
+  name: `ImageFormat`,
+  values: {
+    default: { value: `` },
```

Comment:
@KyleAMathews yeah I thought about a name like that but couldn't come up with one I liked. I considered `original` and `input`. I think `no change` is ok. So that would become NO_CHANGE then?